scripts/drsinter_data_extraction.py

```diff
@@ -367,14 +434,15 @@ def isNaN(string):
 
     fig1, ax1 = plt.subplots()
     for i in range(1, len(cols)):
-        exp_data.plot(ax = ax1, x = 'TIME', y=cols[i], color = color_and_marker[0][0], marker = color_and_marker[0][1], markerfacecolor = color_and_marker[0][2], markevery = int(color_and_marker[0][3]), xlabel='Time (s)', ylabel=(cols[i]+' (' + units[i] + ')'))
+        exp_data.plot(ax=ax1, x='TIME', y=cols[i], color=color_and_marker[0][0], marker=color_and_marker[0][1], markerfacecolor=color_and_marker[0][2], markevery=int(
+            color_and_marker[0][3]), xlabel='Time (s)', ylabel=(cols[i]+' (' + units[i] + ')'))
         ax1.relim()
         ax1.autoscale_view()
         for line in ax1.lines:
-            line.set_marker(None)
+            line.set_marker('')
         ax1.get_legend().remove()
         plt.tight_layout(pad=2)
-        plt.savefig(cols[i] + plot_format)
+        plt.savefig(name_base + cols[i] + plot_format)
         print('Exported', cols[i] + plot_format)
```

Comment:
```suggestion
        print('Exported', name_base + cols[i] + plot_format)
```
In order to match line above.

Reply:
added into the second commit